package.json

```
@@ -49,12 +51,15 @@
     "@babel/eslint-parser": "7.13.4",
     "@babel/preset-env": "7.13.5",
     "@babel/runtime": "7.13.6",
+    "@types/node": "20.8.10",
     "babel-eslint": "10.1.0",
     "eslint": "7.20.0",
     "eslint-plugin-instawork": "file:.",
     "jest": "26.6.3",
     "prettier": "2.2.1",
     "pretty-quick": "3.1.0",
-    "react": "17.0.1"
+    "react": "17.0.1",
+    "ts-node": "^10.9.1",
```

can we use an exact version as for other entries in the file?
```suggestion
    "ts-node": "10.9.1",
```